Moshiur-cy/cleanup-db.js

```diff
@@ -0,0 +1,33 @@
+
+var sql = require("mssql");
+var _ = require('lodash');
+const config = require("./cypress/config/config")

```

Comment:
I did not understand this line...!!!

Reply:
This line actually reads config from a folder location. This is nodejs syntax.

Reply:
Now I understand.